deploy/debian/roles/prepare/tasks/installMaven.yml

```diff
@@ -1,21 +1,21 @@
 ---
 
 - name: Download
-  shell: "curl -O http://mirror.cc.columbia.edu/pub/software/apache/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz"
+  shell: "curl -O https://downloads.apache.org/maven/maven-3/3.8.1/binaries/apache-maven-3.8.1-bin.tar.gz"
```

Comment:
the version number could probably be its own fact. then we could specify it one place, and not have to make changes in multiple places.

im good with approving this though.

Reply:
it might also be worth just removing the version from the path, so that commands can run agnostic of version (at least in terms of where things are).

Reply:
This all makes sense, I'll add the update after I get passing pipelines